src/component/actionbar/index.tsx

```diff
@@ -53,15 +50,13 @@ function ActionBar(props: ActionBarProps) {
     }
 
     selectedTimelines.forEach((timelineId) => {
-      const newUpdate: ICreateUpdate = {
```

Comment:
An example of passing the object directly into the API call instead of creating the object and then passing it.